xbmc/guilib/TextureGL.h

```diff
@@ -9,9 +9,12 @@
 #pragma once
 
 #include "Texture.h"
+#include "guilib/TextureGLFormatMap.h"
 
 #include "system_gl.h"
 
+using namespace KODI::GUILIB::GL;
```

Comment:
```suggestion
namespace KODI::GUILIB::GL {
```

Do we need to use `using` here? Generally, it's a good idea to avoid `using` statements in header files